src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/event/EventHandler.java

```diff
@@ -23,7 +23,7 @@
 	@Bean
 	public Consumer<Message<List<EventGridEvent>>> blobStorageConsumer(Decrypter decrypter,
 			BlobRestConnector blobRestConnector) {
-
+		//Should be wrapped in try-catch in case of malformed blobs' decryption
```

Comment:
@TommasoLencioni Maybe we should add a task in the backlog and remove commented out code